Model/src/GamePackage/Bank.java

```diff
@@ -1,7 +1,158 @@
+     ***************************************************************************************************************/
+    public void receiveHotel() {
+        numHotels++;
+    }
+
+    /****************************************************************************************************************
+     * Returns the numHouses
+     *
+     * @return
+     ***************************************************************************************************************/
+    public int getNumHouses() {
+        return numHouses;
+    }
+
+    /****************************************************************************************************************
+     * Sets the numHouses
+     *
+     * @param numHouses
+     ***************************************************************************************************************/
+    public void setNumHouses(int numHouses) {
+        this.numHouses = numHouses;
+    }
+
+    /****************************************************************************************************************
+     * Returns the number of hotels.
+     *
+     * @return
+     ***************************************************************************************************************/
+    public int getNumHotels() {
+        return numHotels;
+    }
+
+    /****************************************************************************************************************
+     * Sets the number of hotels.
+     *
+     * @param numHotels
+     ***************************************************************************************************************/
+    public void setNumHotels(int numHotels) {
+        this.numHotels = numHotels;
+    }
+
+    /****************************************************************************************************************
+     * Returns the  list of properties owned.
+     *
+     * @return
+     ***************************************************************************************************************/
+    public ArrayList<PropertySquare> getPropertiesOwned() {
+        return propertiesOwned;
+    }
+
+    /****************************************************************************************************************
+     * Sets the current properties owned.
+     *
+     * @param propertiesOwned
+     ***************************************************************************************************************/
+    public void setPropertiesOwned(ArrayList<PropertySquare> propertiesOwned) {
+        this.propertiesOwned = propertiesOwned;
+    }
+
+    /****************************************************************************************************************
+     * Returns the list of mortgaged properties
+     *
+     * @return
+     ***************************************************************************************************************/
+    public ArrayList<PropertySquare> getPropertiesMortgaged() {
+        return propertiesMortgaged;
+    }

```

Comment:
PropertySquares are not the only square that can bemortgaged. This type will also need to change. I think all ownableSquares can also be mortgaged. Another reason to make that change.

Reply:
add this to an issue for the OwnableSquare